### ISSUE TITLE
Duplicated <html> and <body> tags

### ISSUE THREAD

**Issue by andremacola** [RelaxedJS/ReLaXed#111]
When relaxed process the pug or html file, the content generated is duplicated and the DOCTYPE for example is ignored. This can be a problem for custom CSS and JS.

![captura de tela 2018-09-03 as 02 00 30](https://user-images.githubusercontent.com/3408809/44968244-2a191780-af1d-11e8-868c-21690b15386e.png)

**Comment by Zulko** [comment]
Relaxed is not meant to be used on documents with <html> or <body> tags. The head element can be  populated using [a local plugin](https://github.com/RelaxedJS/ReLaXed/wiki/Plugins) (see also [this issue's comments](https://github.com/RelaxedJS/ReLaXed/issues/92)). However it is not possible to give the body a custom CSS class in the current ReLaXed.

**Comment by andremacola** [comment]
@Zulko Right! but I recommend to add standard DOCTYPE as default above <html> because the differences this can cause in the formatting of html and css by chrome. The other stuff like the body classes is not importante in this case.
